lib/active_shipping/carriers/fedex.rb

```diff
@@ -495,15 +500,23 @@ def parse_rate_response(origin, destination, packages, response, options)
                :delivery_range => delivery_range)
         end
 
-        if rate_estimates.empty?
+        if missing_xml_field && logger
+          logger.warn(xml)
+        end
+
```

Comment:
I would `compact` once here on the `rate_estimates` it doesn't have to be repeated.